templates/mks/party_detail.html

```diff
@@ -1,27 +1,28 @@
 {% extends "site_base.html" %}
-{% load i18n links_tags mks_tags common_visual_elements %}{% load static from staticfiles %}
+{% load i18n links_tags mks_tags kikar_tags common_visual_elements %}{% load static from staticfiles %}
 {% block extrajs %}
-    <script src="http://maps.google.com/maps?file=api&amp;v=2&amp;sensor=false&amp;key={{maps_api_key}}" type="text/javascript"></script>
+    <script src="http://maps.google.com/maps?file=api&amp;v=2&amp;sensor=false&amp;key={{ maps_api_key }}"
+            type="text/javascript"></script>
     <script src="{% static "js/mk-pie.js" %}"></script>
     <script type="text/javascript">
 
-        $(function() {
-            var MKPieChart= window.MKPieChart;
+        $(function () {
```

Comment:
@yotammanor  can this "agenda pies" be moved to a seperate js file just like the mk-pie.js above?